litmus-portal/authentication/pkg/utils/configs.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"os"
+	"strconv"
+)
+
+var (
+	JwtSecret         = os.Getenv("JWT_SECRET")
+	AdminName         = os.Getenv("ADMIN_USERNAME")
+	AdminPassword     = os.Getenv("ADMIN_PASSWORD")
+	DBUrl             = os.Getenv("DB_SERVER")
+	DBUser            = os.Getenv("DB_USER")
+	DBPassword        = os.Getenv("DB_PASSWORD")
+	JWTExpiryDuration = getEnvAsInt("JWT_EXPIRY_MINS", 1440)
+	DBName            = "auth"
+	Port              = ":3000"
+	CollectionName    = "users"
+
+	UsernameField = "username"
```

Comment:
Do a ‘gofmt’ and also remove the extra line.

Reply:
Done!